openmicroscopy/openmicroscopy
# develop: Reinstate syncmedia command

This reinstates the syncmedia command. It is useful for people who are deploying with other WSGI servers such as Gunicorn.
## To Test
1. Run `omero web syncmedia` from the command line and it should operate as the initial step of `omero web start`
